receiver/cloudfoundryreceiver/config.go

```diff
@@ -47,6 +48,10 @@ func (c *Config) Validate() error {
 		return err
 	}
 
+	if strings.TrimSpace(c.RLPGateway.ShardID) == "" {
```

Comment:
Is this a breaking change from previous behaviour?

Reply:
Not because of that. The validation was done late in the `stream.go`, we moved to the proper place in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33044/commits/37b090f092468c52928e1cfc8cfe605170f753e3 

The main change regarding the ShardID is because we need to add a suffix `_logs` or `_metrics` when in the past the value was directly used. We have documented this change.

Reply:
ah thanks @jriguera, i missed where it was moved in this diff